blueman/main/Sendto.py

```diff
@@ -204,24 +206,24 @@ def on_transfer_progress(self, _transfer, progress):
 
         self.pb.props.fraction = float(self.total_transferred) / self.total_bytes
 
-    def on_transfer_completed(self, _transfer):
+    def on_transfer_completed(self, _transfer: Transfer) -> None:
         del self.files[-1]
         self.transfer = None
 
         self.process_queue()
 
-    def process_queue(self):
+    def process_queue(self) -> None:
         if len(self.files) > 0:
             self.send_file(self.files[-1].get_path())
         else:
             self.emit("result", True)
 
-    def send_file(self, file_path):
+    def send_file(self, file_path: str) -> None:
         logging.info(file_path)
         if self.object_push:
             self.object_push.send_file(file_path)
 
-    def on_transfer_error(self, _transfer, msg=""):
+    def on_transfer_error(self, _transfer: Optional[Transfer], msg: str = "") -> None:
```

Comment:
Similar to the button case earlier, the signal prohibits the first argument from being None.

Reply:
`on_transfer_failed`